src/vma/proto/dst_entry.h

```diff
@@ -77,15 +77,7 @@ class dst_entry : public cache_observer, public tostr, public neigh_observer
 	in_addr_t	get_dst_addr();
 	uint16_t	get_dst_port();
 	inline in_addr_t get_src_addr() const {
```

Comment:
I don't see why this should be any different than the functions above (get_dst_addr/get_dst_port) with respect to being inline and having the implementation here rather than in the cpp file.